client/state/sites/selectors/is-backup-plugin-active.js

```diff
@@ -8,10 +8,5 @@ import getSiteOption from './get-site-option';
  * @returns {?boolean}        Whether site has the Jetpack Backup plugin active
  */
 export default function isBackupPluginActive( state, siteId ) {
-	const activeJetpackPlugins = getSiteOption( state, siteId, 'jetpack_connection_active_plugins' );
-	if ( activeJetpackPlugins && activeJetpackPlugins.includes( 'jetpack-backup' ) ) {
-		return true;
-	}
-
-	return false;
+	return isJetpackConnectionPluginActive( state, siteId, 'jetpack-plugin' );
```

Comment:
Shouldn't this be `jetpack-backup` instead of `jetpack-plugin`?

Reply:
Woah! Well done.